drivers/ng_at86rf2xx/ng_at86rf2xx_netdev.c

```diff
@@ -293,7 +293,8 @@ static void _receive_data(ng_at86rf2xx_t *dev)
     /* fill missing fields in netif header */
     netif = (ng_netif_hdr_t *)hdr->data;
     netif->if_pid = dev->mac_pid;
-    ng_at86rf2xx_rx_read(dev, &(netif->lqi), 1, pkt_len);
+    /* LQI offset: PHR (1 byte) + Payload (pkt_len) + FCS (2 byte) */
+    ng_at86rf2xx_rx_read(dev, &(netif->lqi), 1, pkt_len + 3);
```

Comment:
this seems still to be wrong: `ng_at86rf2xx_rx_read()` does not return the PHR. So shouldn't  `ng_at86rf2xx_rx_read(dev, &(netif->lqi), 1, pkt_len + 2)` be the correct solution?